app/gallery/models/picture.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from django.db.models.fields.related import ForeignKey
+
+from app.common.enums import AdminGroup
+from app.common.permissions import BasePermissionModel
+from app.content.models.event import Event
+from app.util.models import BaseModel
+
+
+class Picture(BaseModel, BasePermissionModel):
+
```

Comment:
primary key som slug? elller sluggify tittelen for filterering eller skjer det automatisk når man bruker det som look up field

Reply:
ganske sikker på at det skjer automatisk med lookupfield siden det fungerte bra. Hadde en del annen kode hvor jeg satte tittel som slug, men det viste seg å ikke legge til noe annen funksjonalitet.

Reply:
klarer man å finne flere bilder med samme tittel med lookup_field, eller vil den bare ta det første bildet? 🤔

Reply:
fant  ut at sånn som det er nå, vil å lete etter to bilder med samme tittel ikke fungere

Reply:
Me burde gjøre sånn at tittelen er unik, og at sluggen blir generert basert på den. I tillegg så bør sluggen være PK for bildet

Reply:
to muligheter å implementere. 1. vi lager en uuid for hvert bilde. 2. hvis et bilde ser at det finnes et annet bilde med samme slug legger den til et tall på slutten. som: bilde, bilde-2, bilde-3

Reply:
Støtter her å ha en UUID som PK. Det er ikke alltid nødvendig med en tittel/slug: se for dere at man laster opp 200 bilder fra et arrangement - da er det ikke mange som kommer til å gå gjennom alle bildene og legge inn tittel.

Reply:
tenker du da at det er en default tittel? Eller at blank = true? Eventuelt ikke tittel i det hele tatt?